### PR TITLE
avoid reparse json string when loading into json type

### PR DESCRIPTION
**Pull request by murphyatwork**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] others

## Motivation
Avoid parse json twice when loading json file into json column.

## Modification
Before this, loading json data into json column needs parsing json twice, `simdjson` parse and `velocypack` parse.

After this, simdjson::object could be serialized into binary format directly, avoid parsing twice.

### CODE REVIEW

**Review by kangkaisen — DISMISSED** [review]

File: be/src/util/json.cpp [hunk 1]
```diff
@@ -63,67 +63,167 @@ JsonValue JsonValue::from_string(const Slice& value) {
     return JsonValue(builder.slice());
 }
 
-StatusOr<JsonValue> JsonValue::from_simdjson(simdjson::ondemand::value* value) {
-    namespace sj = simdjson::ondemand;
+namespace so = simdjson::ondemand;
```
Comment (satanson) [inline be/src/util/json.cpp:66]: suggest to use Camel name of  type aliases.

File: be/src/util/json.cpp [hunk 2]
```diff
@@ -63,67 +63,167 @@ JsonValue JsonValue::from_string(const Slice& value) {
...
+
+// Convert SIMD-JSON object to a JsonValue
+class SimdJsonConverter {
+public:
+    static StatusOr<JsonValue> create(so::value value) {
```
Comment (satanson) [inline be/src/util/json.cpp:71]: for type so::value, suggest to use:
```
using SmidJsonValue = smidjson::ondemand::value;
```

Comment (mofeiatwork) [reply be/src/util/json.cpp:71]: done.

File: be/src/util/json.cpp (hunk 1 above)
Comment (mofeiatwork) [reply be/src/util/json.cpp:66]: namespace should observe lower-case naming rule?

**Review by kangkaisen — APPROVED** [review]

**Review by satanson — APPROVED** [review]

**Pull request merged**